misc/python/materialize/cli/run.py

```diff
@@ -495,7 +495,13 @@ def _cargo_build(
     programs = [*REQUIRED_SERVICES, *extra_programs]
     for program in programs:
         command += ["--bin", program]
-    completed_proc = spawn.runv(command, env=env)
+    completed_proc = spawn.runv(
+        command,
+        env=env,
+        cwd=pathlib.Path(
+            os.path.abspath(os.path.join(os.path.realpath(sys.argv[0]), "../.."))
+        ),
```

Comment:
Nit: If you construct the `Path` object first, this becomes a bit nicer. Not tested, but this should be equivalent:

```
cwd=Path(sys.argv[0]).resolve().parents[1]
```